spec/suma/async/jobs_spec.rb

```diff
@@ -206,6 +206,32 @@
       end.to perform_async_job(Suma::Async::ResetCodeUpdateTwilio)
     end
 
+    it "ignores twilio 404s for invalid, expired or approved reset codes" do
```

Comment:
These should be tested in `twilio_spec`, not the job.

Reply:
Thanks, I had suspicion, they're moved now